### DISCUSSION TITLE
My stats are completely wrongs

### DISCUSSION THREAD

**Discussion by Diegojacober** [anuraghazra/github-readme-stats#3306] [Q&A] [answered]
Hi, thanks for this tool, is amazing for all the community.

i have sure that my stats are incorrects!

https://github-readme-stats.vercel.app/api?username=diegojacober&show_icons=true&theme=radical
in this card, it's showing that i have 73 commits in this year, but only in one repository(https://github.com/Diegojacober/curso-python)  there are more than 100 commits.

Can someone help me? It's just my studies, but I'd like to see my stats

**Comment by qwerty541** [comment, marked as answer]
Hey, @Diegojacober! I checked repository that you provided, if you filter the commits in this repository by authorship, you will find that it contains only 3 commits connected to your GitHub account at the moment. You can check it yourself by the following link:

https://github.com/Diegojacober/curso-python/commits?author=Diegojacober

So stats card shows correct data, we fetch it directly from GitHub API. I assume that there is some troubles with git config on your working PC. You can try to set same email and username as in your GItHub account following instructions in this article: 

https://support.atlassian.com/bitbucket-cloud/docs/configure-your-dvcs-username-for-commits/
